### PR TITLE
fix(api): get subscriber preference parameter swagger value

### PR DESCRIPTION
**Pull request by jainpawan21**
### What changed? Why was the change needed?

Replace `TriggerRecipientsTypeEnum` with `PreferenceLevelEnum`
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/api/src/app/subscribers/subscribers.controller.ts
Comment (SokratisVidros) [inline apps/api/src/app/subscribers/subscribers.controller.ts:425]: What does the parentheses mean?

Comment (jainpawan21) [reply apps/api/src/app/subscribers/subscribers.controller.ts:425]: valid values can be template or global

Comment (SokratisVidros) [reply apps/api/src/app/subscribers/subscribers.controller.ts:425]:
```suggestion
    description: 'Retrieve global or workflow level subscriber preferences',
```

**Review by SokratisVidros — APPROVED** [review]

File: apps/api/src/app/subscribers/subscribers.controller.ts
Comment (jainpawan21) [reply apps/api/src/app/subscribers/subscribers.controller.ts:425]: Will fix this in next PR, this PR got auto merged after approval

**Pull request merged**
